doc/conf.py

```diff
@@ -380,6 +380,7 @@
     ('boards/arm/olimex_stm32_p405/doc/olimex_stm32_p405', 'boards/arm/olimex_stm32_p405/doc/index'),
     ('boards/arm/olimexino_stm32/doc/olimexino_stm32', 'boards/arm/olimexino_stm32/doc/index'),
     ('boards/arm/qemu_cortex_m3/doc/board', 'boards/arm/qemu_cortex_m3/doc/index'),
+    ('boards/arm/rak5010_nrf52840/doc/rak5010_nrf52840', 'boards/arm/rak5010_nrf52840/doc/index'),
```

Comment:
This is not needed ,since we're adding a new platform. No need for redirect page.